src/Halogen/Store/Monad.purs

```diff
@@ -70,6 +67,23 @@ derive newtype instance MonadWriter w m => MonadWriter w (StoreT a s m)
 derive newtype instance MonadState s m => MonadState s (StoreT a s m)
 derive newtype instance MonadCont m => MonadCont (StoreT a s m)
 derive newtype instance MonadRec m => MonadRec (StoreT a s m)
+derive newtype instance MonadFork f m => MonadFork f (StoreT a s m)
+
+-- Can't use generalized newtype deriving because it produces unverifiable
+-- superclass constraint warnings
+instance MonadKill e f m => MonadKill e f (StoreT a s m) where
+  kill e = lift <<< kill e
+
+-- Can't use generalized newtype deriving because it produces unverifiable
+-- superclass constraint warnings
+instance MonadBracket e f m => MonadBracket e f (StoreT a s m) where
```

Comment:
I'm curious what you're using this for. Could you give me an example?

Reply:
Sure - I don't have a specific use for it as such, but I have cases where I want to work in a context that provides access to `MonadBracket m`. If my concrete monad has a `MonadStore` in it, I want to be able to use `derive newtype ...` to derive these classes for it, and `StoreT` supports it naturally, so it's useful to provide it.

e.g.

```purescript
newtype AppM = AppM (StoreT Action Store (ReaderT AppEnv Aff))

... derive required superclasses
derive newtype instance MonadFork Fiber AppM
derive newtype instance MonadKill Error Fiber AppM
derive newtype instance MonadBracket Error Fiber AppM
```